lib/exhal/assertions.ex

```diff
@@ -67,6 +67,13 @@ defmodule ExHal.Assertions do
   end
 
   @doc"""
+    Checks if a link is present in a rendered document.
+  """
+  def has_link?(json_map, rel) do
+    Kernel.get_in(json_map, ["_links", rel])
```

Comment:
can we do it like https://github.com/pezra/exhal/pull/48/files#diff-1dfa1b718cc1f0397429a38b9f1be695R126 does? This approach will fail in some situations when it should not.